src/main/minio.js

```diff
@@ -1488,7 +1490,7 @@ export class Client {
       throw new TypeError('expires should be of type "number"')
     }
     var method = 'PUT'
-    var requestDate = Moment().utc()
+    var requestDate = new Date()
```

Comment:
new Date() does not return UTC time when run in browser (minio-js can be run in browser as well)